src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj

```diff
@@ -47,14 +47,15 @@
   <ItemGroup>
     <PackageReference Include="Arcus.Messaging.Health" Version="0.1.0" />
     <PackageReference Include="Arcus.Messaging.Pumps.ServiceBus" Version="0.1.0" />
+    <PackageReference Include="Arcus.Observability.Telemetry.Core" Version="0.3.0" Condition="'$(ExcludeSerilog)' == 'false'" />
```

Comment:
Well you can even use the extensions without Serilog, it's not a hard requirement

Reply:
Ok, I'll remove the conditions.